public/components/marketing-buttons/16.html

```diff
@@ -0,0 +1,5 @@
+<div class="relative"> 
+  <a class="font-bold text-base text-primary text-black after:content-[''] after:absolute after:-bottom-2 after:h-[2px] after:w-0 after:left-1/2 after:bg-black after:transition-all after:duration-300 after:block hover:after:transition-all hover:after:w-full hover:after:left-0 hover:after:duration-300 hover:after:block" href="/download">
```

Comment:
`test-base text-black after:content-['']` aren't needed, they are applied by default.

There seem to be a lot of classes on the `before` which feels overkill, here's what I created - https://play.tailwindcss.com/AgRmlrUbo5

Reply:
Hi, @markmead already refactored the code and add some new variations. :))